java/java-impl-refactorings/src/com/intellij/refactoring/anonymousToInner/AnonymousToInnerHandler.java

```diff
@@ -688,43 +688,18 @@ private void createSuperStatement(PsiMethod constructor, PsiExpression[] paramEx
     PsiMethodCallExpression methodCall = (PsiMethodCallExpression) ((PsiExpressionStatement) statement).getExpression();
     PsiExpressionList exprList = methodCall.getArgumentList();
 
-
-    {
-      final PsiThisExpression qualifiedThis =
-        (PsiThisExpression) factory.createExpressionFromText("A.this", null);
-      final PsiJavaCodeReferenceElement targetClassRef = factory.createClassReferenceElement(myTargetClass);
-      PsiJavaCodeReferenceElement thisQualifier = qualifiedThis.getQualifier();
-      assert thisQualifier != null;
-      thisQualifier.replace(targetClassRef);
-
-      for (PsiExpression expr : paramExpressions) {
-        ChangeContextUtil.encodeContextInfo(expr, true);
-        final PsiElement newExpr = exprList.add(expr);
-        ChangeContextUtil.decodeContextInfo(newExpr, myTargetClass, qualifiedThis);
-      }
-    }
-
-    class SupersConvertor extends JavaRecursiveElementVisitor {
-      @Override public void visitThisExpression(@NotNull PsiThisExpression expression) {
-        try {
-          final PsiThisExpression qualifiedThis =
-                  (PsiThisExpression) factory.createExpressionFromText("A.this", null);
-          final PsiJavaCodeReferenceElement targetClassRef = factory.createClassReferenceElement(myTargetClass);
-          PsiJavaCodeReferenceElement thisQualifier = qualifiedThis.getQualifier();
-          assert thisQualifier != null;
-          thisQualifier.replace(targetClassRef);
-          expression.replace(qualifiedThis);
-        } catch (IncorrectOperationException e) {
-          LOG.error(e);
-        }
-      }
-
-      @Override public void visitReferenceExpression(@NotNull PsiReferenceExpression expression) {
-      }
+    final PsiThisExpression qualifiedThis =
+      (PsiThisExpression) factory.createExpressionFromText("A.this", null);
```

Comment:
I think you can use `var` here to not write `PsiThisExpression` twice. Maybe then it will fit in a single line.

Reply:
Or if it already firs in a single line, just reformat and leave it like this